test/integration/error.test.js

```diff
@@ -153,7 +153,7 @@ describe(Support.getTestDialectTeaser('Sequelize Errors'), () => {
     });
 
     it('SequelizeValidationErrorItem.Origins is valid', () => {
-      const ORIGINS = Sequelize.ValidationErrorItem.Origins;
+      const ORIGINS = Sequelize.ValidationErrorItemOrigin;
```

Comment:
Does this exists?

Reply:
I found it further up. I tend to think that this might be breaking change and that we should add an alias + remove it in v7?

Reply:
Setting it as static property of `ValidationErrorItem` now, added the old test back, next to this one and added the deprecation flag